engine/Assets/Scripts/Modes/Aether/ServerTestMode.cs

```diff
@@ -39,7 +62,37 @@ public class ServerTestMode : IMode {
         }
     }
 
-    public void Update() {}
+    public void Update() {
+        var transformsList = new List<ServerTransforms>();
+        if (_ghost is null || _host is null)
+            return;
+        if (_ghost.Client is null || _host.Client is null)
+            return;
+
```

Comment:
There are quite a few nullable reference type warnings around here that should probably be addressed when we integrate multiplayer into match mode.